browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -2085,6 +2085,15 @@ Are you sure you want to do this?
     <message name="IDS_RETYPE_PASSWORD_ERROR" desc="Brave Wallet password error text">
       Verified password did not match
     </message>
+    <message name="IDS_CREATING_WALLET_ONBOARDING_TEXT" desc="Text displayed along with a spinner during Brave Wallet creation or restoring.">
+      Creating Wallet...
+    </message>
+    <message name="IDS_YOUR_WALLET_IS_RESTORING_PAGE_TITLE" desc="Text for a spinner page title during Brave Wallet restoring. It is not displayed anywhere" translateable="false">
+      Your wallet is restoring...
+    </message>
+    <message name="IDS_YOUR_WALLET_IS_CREATING_PAGE_TITLE" desc="Text for a spinner page title during Brave Wallet creating. It is not displayed anywhere" translateable="false">
+      Your wallet is creating...
```

Comment:
Suggestions: `Your wallet is being created...` or `Creating your wallet...`

Reply:
Changed to `Your wallet is being created...`, thanks.